src/aircloak/AircloakApi.cs

```diff
@@ -0,0 +1,363 @@
+using System;
+using System.Net.Http;
+using System.Net;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Net.Http.Headers;
+
+using Explorer.Api.AircloakApi.ReponseTypes;
+using Explorer.Api.AircloakApi.Helpers;
+
+namespace Explorer.Api.AircloakApi
+{
+    using DataSources = List<DataSource>;
+
+    /// <summary>
+    /// Contains the HttpClient instance and doles out ApiSession objects.
+    /// </summary>
+    public static class SessionManager
+    {
+        static private readonly AircloakApiClient ApiClient = new AircloakApiClient();
+
+        public static AircloakApiSession NewSession(string apiRootUrl, string apiKey)
+        {
+            return new AircloakApiSession(ApiClient, apiRootUrl, apiKey);
+        }
+    }
+
+    /// <summary>
+    /// Provides higher-level access to the Aircloak Http Api. The session uses the same Url root and Api Key 
+    /// throughout its lifetime. 
+    /// </summary>
+    /// <see cref=AircloakApiClient/>
+    public class AircloakApiSession
```

Comment:
You're right, I wasn't totally consistent here, but I think it's good to keep a prefix on some classes otherwise the name becomes too general. Also, it might clash with something else imported from another package. 

This seems to be quite common in the .NET core api. For example look at the `System.Text.Json` namespace, almost all the class names have `Json` as a prefix. 

Let me know what you think... (@sebastian, any opinion on this?).

Reply:
I have no opinion. I'll leave this to the two of you

Reply:
I agree, in this situation it is difficult to find better names.

Reply:
I'm going to reorganise the namespaces and will figure out a better naming scheme.